### DISCUSSION TITLE
关于siamRPN训练问题

### DISCUSSION THREAD

**Discussion by cvzhang** [STVIR/pysot#530] [Code]
你好，开始训练backbone时就开始CUDA out of memory，也就是第十个epoch后出现爆显存，显存只有8G，如果把start epoch改成10也会出现这种情况，请问需要修改哪里进行下一步训练吗？

**Comment by ZhiyuanChen** [comment]
这是因为前10个epoch处于warmup阶段，backbone不参与训练。
将Batch Size调低可以缓解这个问题

**Reply by cvzhang** [reply]
已解决，谢谢
